assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -124,6 +138,61 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
     [rgb, onChange]
   );
 
+  const {
```

Comment:
Maybe move the eyedropper into a separate file? Sounds like a feature that would make sense living on its own.

Reply:
Moved [here](https://github.com/google/web-stories-wp/pull/7961/commits/2f96e02abb4e29fc99bd695228008c098af63645), I was also thinking about lazy loading that part (example in commit), WDYT?